check_mk_web_api/test_WebApi.py

```diff
@@ -530,12 +530,18 @@ def test_logout_site():
     api.logout_site('cmk')
 
 def test_bulk_discovery_start():
+    if 'CHECK_MK_VERSION' in os.environ and os.environ['CHECK_MK_VERSION'] == "1.5":
+        # only supported since version 1.6
+        return
     while api.bulk_discovery_status()['is_active'] == True:
         time.sleep(5)
     api.add_host('host00')
     api.bulk_discovery_start(['host00'], mode=WebApi.DiscoverMode.NEW)
 
 def test_bulk_discovery_all_hosts():
+    if 'CHECK_MK_VERSION' in os.environ and os.environ['CHECK_MK_VERSION'] == "1.5":
+        # only supported since version 1.6
+        return
```

Comment:
Please use the pytest.skip function instead of just returning.

Reply:
Done in bcdcab1fd5fb337ef4179280eb6dda920867ac29